bundle.Dockerfile

```diff
@@ -1,8 +1,15 @@
 FROM scratch
 
-# These directories map to the layout of the directories in the WMCO dist-git repository.
-COPY manifests /manifests/
-COPY metadata/annotations.yaml /metadata/annotations.yaml
+# This block are standard Red Hat container labels
+LABEL name="openshift4/windows-machine-config-operator-bundle" \
+    License="ASL 2.0" \
+    io.k8s.display-name="Windows Machine Config Operator bundle" \
+    io.k8s.description="Windows Machine Config Operator's OLM bundle image" \
+    summary="Windows Machine Config Operator's OLM bundle image" \
+    maintainer="Team Windows Containers <team-winc@redhat.com>"
+
```

Comment:
Please move the labels to one spot. I would rather not have the labels spread around.